podpac/datalib/egi.py

```diff
@@ -229,8 +227,6 @@ def eval(self, coordinates, output=None):
                 " which case EGI is returning no data."
             )
             raise e
-        # Force update on native_coordinates (in case of multiple evals)
-        self.set_trait("native_coordinates", self.get_native_coordinates())
```

Comment:
@mpu-creare The `native_coordinates` are now a property, so I just removed this `set_trait` call. Is that okay?

Originally I made the native_coordinates a cached_property, but this `set_trait` suggests to me that we shouldn't be saving anything between evals, so I changed it to just a normal property.

Reply:
I'm not sure @mlshapiro can you advise?